### DISCUSSION TITLE
Allow a Shop pack to be activated/de-activated in the Shop packs list

### DISCUSSION THREAD

**Discussion by babsgosgens** [gameserverapp/Platform#900] [Suggestions]
In the Shop pack list, one of these user actions could enable/disable a Shop pack:

1. Click the `hidden` or `active` label (use it as a toggle)
2. Add an option to the pulldown menu to enable/disable

![image](https://github.com/user-attachments/assets/72ce67a3-2789-4756-9011-477026a858d1)

**Comment by mvaessen** [comment]
This will be available in the next update, thanks for the suggestion!

**Comment by babsgosgens** [comment]
Happy Goose. Thank you!
